modules/infrastructure-node-sizing.adoc

```diff
@@ -7,21 +7,53 @@
 
 The infrastructure node resource requirements depend on the cluster age, nodes, and objects in the cluster, as these factors can lead to an increase in the number of metrics or time series in Prometheus. The following infrastructure node size recommendations are based on the results of cluster maximums and control plane density focused testing.
 
+[IMPORTANT]
+====
+The sizing recommendations below are only applicable for the Prometheus, Router, and Registry infrastructure components, which are installed during cluster installation. Logging is a day-two operation and is not part of these recommendations.
+====
+
 [options="header",cols="3*"]
 |===
 | Number of worker nodes |CPU cores |Memory (GB)
 
 | 25
 | 4
-| 32
+| 16
 
 | 100
 | 8
-| 64
+| 32
 
 | 250
+| 16
+| 128
+
+| 500
 | 32
-| 192
+| 128
+
```

Comment:
Sorry to be nit-picky but I don't like the two tables back-to-back with no description of the second table.   @ahardin-rh feel free to overrule me on this.   In my opinion, a better approach might be a single table where the core and memory columns have two values:  one without logging and one with logging.    Opinions everyone?

Reply:
That seems like a good suggestion to me!